assemblies/configuring-trustyai.adoc

```diff
@@ -2,16 +2,17 @@
 
 ifdef::context[:parent-context: {context}]
 
-[id="setting-up-model-monitoring_{context}"]
-= Setting up model monitoring
+[id="configuring-trustyai_{context}"]
+= Configuring TrustyAI
 
 [role='_abstract']
 
-To set up model monitoring for data scientists to use in {productname-short}, a cluster administrator does the following tasks:
+To configure model monitoring with TrustyAI for data scientists to use in {productname-short}, a cluster administrator does the following tasks:
```

Comment:
If this module is aimed at cluster administrators, could we not use "you" instead of "cluster administrator"?

..."you must complete the following tasks".

Reply:
I agree this sounds strange, but I wanted to make it clear this section was for admins until we separate the books.